assets/css/style.css

```diff
@@ -512,20 +544,28 @@ span.rotated {
 }
 
 .percentAreaChart line {
-  stroke: black;
-  stroke-width: 1;
   stroke-dasharray: 5;
-  opacity: 0.5;
```

Comment:
I'd prefer to either leave the opacity or change the stroke-width to 0.5. I think the lighter lines look much better on smaller screens, and no worse on larger screens